vm/systemSmartContracts/delegation.go

```diff
@@ -1887,9 +1887,9 @@ func (d *delegation) getFund(key []byte) (*Fund, error) {
 	return dFund, nil
 }
 
-func (d *delegation) createAndSaveNextFund(address []byte, value *big.Int, fundType uint32) ([]byte, error) {
+func (d *delegation) createAndSaveNexKeyFund(address []byte, value *big.Int, fundType uint32) ([]byte, error) {
```

Comment:
createAndSaveNextKeyFund ? however KeyFund does not sound so right. and saveKeyFund might be renamed again to saveFund - as it only safe the fund data.

Reply:
Done.